src/components/UI/Card.svelte

```diff
@@ -0,0 +1,125 @@
+<script>
+  import { _ } from 'svelte-i18n';
+  import Text from './Text.svelte';
+  import Icon from './Icon.svelte';
+  import Image from './Image.svelte';
+  import { calendarIcon, clockIcon, arrowIcon } from '@/images/icons';
+
+  export let languageAbbreviation = 'EN';
+  export let title = '';
+  export let src = '';
+  export let date = '';
+  export let time = '';
+  export let href = '';
+  export let group = null;
+
+  const openLinkInNewTab = () => {
+    window.open(href, '_blank');
+  };
+</script>
+
+<div class="card" on:click={openLinkInNewTab}>
+  <div class="badges">
+    <div class="badge">{languageAbbreviation}</div>
+    {#if group}
+      <div class="badge">{group}</div>
+    {/if}
+  </div>
+  <div class="images">
+    <Image {src} alt={title} rounded={false} />
+  </div>
+  <div class="when">
+    <div class="date">
+      <div class="icon">
+        <Icon icon={calendarIcon} />
+      </div>
+      <Text size="m">{date}</Text>
+    </div>
+    <div class="time">
+      <div class="icon">
+        <Icon icon={clockIcon} />
+      </div>
+      <Text size="m">{time}</Text>
+    </div>
+  </div>
+  <div class="title">
+    <Text size="l" weight="bold">{title}</Text>
+  </div>
+  <span class="filler" />
```

Comment:
Nasty, you should be able to do this spacing with css, without adding this as an element. If this is too much of a css rework for the time available, please hide it from assistive technology with `aria-hidden="true"`
```suggestion
  <span aria-hidden="true" class="filler" />
```